packages/ui-extensions-react/src/surfaces/customer-account/components/PolicyModal/PolicyModal.ts

```diff
@@ -27,4 +27,7 @@ export interface PolicyModalProps {
   type: PolicyType;
 }
 
-export const PolicyModal = createRemoteReactComponent('PolicyModal');
+export const PolicyModal = createRemoteReactComponent<
```

Comment:
I have honestly no idea how/why this worked before 🤔
We never combined the Component with its types. We're now passing the props so that the type is complete.